core/platform/search/elastic_search_services.py

```diff
@@ -118,12 +116,11 @@ def search(
     on gae_search_services.py is removed from the codebase.
 
     Args:
-        query_string: str. A JSON-encoded string representation of the
-            dictionary search definition that uses Query DSL. See
-            elastic.co/guide/en/elasticsearch/reference/current/query-dsl.html
-            for more details about Query DSL.
+        query_string: str. The terms that the user is searching for.
         index_name: str. The name of the index. Use '_all' or empty string to
             perform the operation on all indices.
+        categories: list(str). The list of categories to query for.
+        language_codes: list(str). The list of language codes to query for.
```

Comment:
The docstring should explain that the query will be made using "OR" logic for languagecode and categories, right?

Reply:
Done, good catch!